docs/getting-started/quickstart.md

```
@@ -1,55 +1,139 @@
+	>
+	> If you prefer to skip docstrings for obvious utility functions in this project, that's perfectly reasonable. Would you like me to adjust my review approach regarding documentation for this repository?
+
+	In this case, CodeRabbit is offering to adjust its future behavior when reviewing changes to this repository. Instead, follow the next step to ask CodeRabbit to implement part of its suggestions.
+
+1. Post the following as another new comment:
+
+	```
+	@coderabbitai generate docstrings
+	```
+
+	After a few moments, CodeRabbit does the following:
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)